platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -101,7 +113,11 @@ protected void initView() {
 	}
 
 	public boolean isMultiline() {
-		return mMultiline;
+		return mKeyboardType == VirtualKeyboardType.KEYBOARD_TYPE_MULTILINE;
+	}
```

Comment:
I think you could remove `isMultiline`. It was only used twice, you removed one use and the other one is in:
```
platform/android/java/lib/src/org/godotengine/godot/input/GodotTextInputWrapper.java
118:                    if ((key == '\n') && !mEdit.isMultiline()) {
```
where it could use `getKeyboardType` I suppose.

Reply:
This is true. Though, I think a case could be made regarding compatibility if any java plugins are using `isMultiline` for anything. I have no opinion either way, `isMultiline` can be easily removed.

Reply:
That's a good point, let's keep it in `3.x` then. It could be removed in `master`.